ministryofjustice/laa-review-criminal-legal-aid
# Crimapp 231 add reviewed on to read model

## Description of change
add reviewed_on to Review read model

## Link to relevant ticket
[CRIMAPP-231](https://dsdmoj.atlassian.net/browse/CRIMAPP-231)

## Notes for reviewer

This is in preparation for using the read mode of the processed mini report.

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature

Run the migration and make sure the read model for a reviewed application has its reviewed_on date stored.

[CRIMAPP-231]: https://dsdmoj.atlassian.net/browse/CRIMAPP-231?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
